### ISSUE TITLE
Configure labels individually

### ISSUE THREAD

**Issue by darklight721** [AnyChart/AnyChart#7]
Is there a way to configure labels individually?

I wanted to do something like this:

```javascript
var labels = chart.xAxis().labels();
labels.getLabel(0).background('#f00');
```

I see that there is a `getLabel` method on `anychart.core.ui.LabelsFactory.prototype`, but somehow that's not available in `labels`.

**Comment by loki20beta** [comment]
@darklight721  Are you looking for background specifically? There is a way to work with labels text using formatter like this: http://jsfiddle.net/gc8wfzwy/1/

**Comment by darklight721** [comment]
@loki20beta Yes, I actually wanted to attach an image to a label. I could do that with `labels().background().fill({ src: 'some_image.jpg' })`, but that would set it to all labels. I want them to have different images.

**Comment by loki20beta** [comment]
@darklight721 okay, I see. We'll dig into this and let you know if it is possible or when it will be possible on Monday.

**Comment by darklight721** [comment]
@loki20beta alright, thanks!

**Comment by loki20beta** [comment]
@darklight721 unfortunately, it is not possible at the moment, you can add individual picture only to the labels on the series. We feel that it is an omission and will put a request for improvement. Dev lead will assign a version where it will be fixed later.

**Comment by loki20beta** [comment]
@darklight721 since version 7.13.1 you can do this like this: http://jsfiddle.net/gc8wfzwy/5/

we will try to make it more convenient in future versions

**Comment by darklight721** [comment]
@loki20beta It works now, thanks!

**Comment by maresha** [comment]
Please check http://jsfiddle.net/gc8wfzwy/7/ 
We implemented this feature with 7.14.0 version, not it has to be enough. Please try it and let us know if it suits your needs.

**Comment by maresha** [comment]
Please check http://jsfiddle.net/gc8wfzwy/7/ 
We implemented this feature with 7.14.0 version, not it has to be enough. Please try it and let us know if it suits your needs.
